examples/seismic/tutorials/09_Snapshots_and_ConditionalDimension.ipynb

```diff
@@ -0,0 +1,830 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Snapshotting with Devito and the `ConditionalDimension` function\n",
```

Comment:
Search for all occurrences of "`ConditionalDimension` function" (I spotted a few): `ConditionalDimension` is not a function. I'd just use `ConditionalDimension`; for example, here:

"# Snapshotting in Devito with `ConditionalDimension`\n"

Reply:
Fixed.